vsc_equiv.erl

```diff
@@ -47,47 +47,54 @@ check_equiv(OrigHash, RefacHash, OrigName, RefacName) ->
     % This is needed because QuickCheck has to evaluate to old and the new
     % function repeatedly side-by-side
     checkout(OrigHash),
-    comp(OrigName, ProjFolder, "old"),
+    comp(OrigName, ProjFolder, "orig"),
 
     checkout(RefacHash),
-    comp(RefacName, ProjFolder, "new"),
-    
-    % Save the current module path, and create two new module paths
-    % with the folder for the original and the refactored modules added
-    ModulePath = code:get_path(),
-    OldModules = ["old"] ++ ModulePath,
-    NewModules = ["new"] ++ ModulePath,
-    
+    comp(RefacName, ProjFolder, "refac"),
+
+    % Hard-coded for now
+    start_nodes(),
+    OrigNode = 'orig@x200s',
+    RefacNode = 'refac@x200s',
+
     % Generate random data (assume we know the type of data for now) and check
     % if the outputs are equivalent
-    % RES = proper:quickcheck(?FORALL(I, integer(), prop_same_output(OldModules, NewModules, test, f_old, f_new, [I]))), % TODO Give back the minimal input that falsifies the assertion
-    RES = proper:quickcheck(?FORALL(L, list(integer()), prop_same_output(OldModules, NewModules, test, f_old, f_new, [L]))), % TODO Give back the minimal input that falsifies the assertion
+    % RES = proper:quickcheck(?FORALL(I, integer(), prop_same_output(OrigNode, RefacNode, test, f_old, f_new, [I]))), % TODO Give back the minimal input that falsifies the assertion
+    RES = proper:quickcheck(?FORALL(L, list(integer()), prop_same_output(OrigNode, RefacNode, test, f_old, f_new, [L]))), % TODO Give back the minimal input that falsifies the assertion
 
     file:set_cwd(".."),
     cleanup(),
-    code:set_path(ModulePath),
     application:stop(wrangler), % TODO
     RES.
 
-prop_same_output(OldPath, NewPath, Module, OldName, NewName, Args) ->
-    % Gets the module paths with the respective folders added (old and new)
-    % Also gets the module name and the original and refactored function names
-    % and the input used to evaluate them
-    change_module(OldPath, Module),
-    A = erlang:apply(Module, OldName, Args),
+start_nodes() ->
```

Comment:
Is there a better way to start a new erlang VM?
For now I'm just spawning a new OS process, but other than being very hacky, I'm also not able to close it afterwards, as I'm not getting back the Pid of the new process.

Reply:
Try https://www.erlang.org/doc/man/slave.html

Reply:
I've used the peer module: https://www.erlang.org/doc/man/peer.html.